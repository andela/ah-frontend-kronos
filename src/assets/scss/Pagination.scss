.btn-pagination{
    margin: 2% 40%;
    width: 100px;
}

button[name=previous], button[name=next]{
    display: inline-block;
    color: white;
    background-color: #023873;
    height: 5vh;
}


span{
  margin: 0% 50%;
  color: #023873;
}