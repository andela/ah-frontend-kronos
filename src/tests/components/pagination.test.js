import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';

import Pagination from '../../components/articles/pagination';

describe('<Pagination />', () => {
  const handlePagination = () => {};
  const wrapper = shallow(<Pagination
    currentPage={1}
    totalNumberOfPages={1}
    handlePagination={handlePagination}
  />);

  it('matches snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
