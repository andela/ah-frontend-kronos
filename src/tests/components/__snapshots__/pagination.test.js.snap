// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Pagination /> matches snapshot 1`] = `
<div
  className="pagination"
>
  <button
    disabled={true}
    name="previous"
    onClick={[Function]}
    type="button"
  >
    Previous
  </button>
  <span>
    1/1
      pages
  </span>
  <button
    disabled={true}
    name="next"
    onClick={[Function]}
    type="button"
  >
    Next
  </button>
</div>
`;
